src/Files.App/UserControls/Widgets/QuickAccessWidget.xaml

```diff
@@ -72,10 +72,29 @@
 											<FontIcon Glyph="&#xE737;" />
 										</MenuFlyoutItem.Icon>
 									</MenuFlyoutItem>
+									<MenuFlyoutItem
+										x:Name="PinToFavorites"
+										Click="PinToFavorites_Click"
+										DataContext="{x:Bind}"
+										Text="{helpers:ResourceString Name=BaseLayoutItemContextFlyoutPinToFavorites/Text}"
+										Visibility="Collapsed">
+										<MenuFlyoutItem.Icon>
+											<FontIcon Glyph="&#xE840;" />
+										</MenuFlyoutItem.Icon>
+									</MenuFlyoutItem>
+									<MenuFlyoutItem
+										x:Name="UnpinFromFavorites"
+										Click="UnpinFromFavorites_Click"
+										DataContext="{x:Bind}"
+										Text="{helpers:ResourceString Name=BaseLayoutContextFlyoutUnpinFromFavorites/Text}"
+										Visibility="Collapsed">
+										<MenuFlyoutItem.Icon>
+											<FontIcon Glyph="&#xE77A;" />
+										</MenuFlyoutItem.Icon>
+									</MenuFlyoutItem>
```

Comment:
Should there be an option to remove a folder from quick access?

Reply:
I think there should, it only shows up for the pinned items

Reply:
I think it should only be there for recent folders, will look into it

Reply:
Agreed